dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -42,8 +36,8 @@ export const NoExpiringRuns = () => {
         <Text component={TextVariants.h3}> You have no runs expiring soon</Text>
         <Text component={TextVariants.p}>
           Runs that have expiration date within next {EXPIRATION_DAYS_LIMIT}{" "}
-          days will appear here. These runs will be automatically removed from
-          the system if left unacknowledged.{" "}
+          days will appear here. These runs will be automatically removed from{" "}
+          the system if left unacknowledged.
           <Button variant="link">Learn More.</Button>
         </Text>
```

Comment:
I hate to even ask, but....

When this text component is rendered, the text is not copied literally (with just the `{}` substitutions replaced) -- it is rewrapped (or "reflowed" or whatever the term is) and the line breaks and indentation are removed...right?  Are you saying that, e.g., without the `{" "}` at the end of line 39, the `from` there would be concatenated to the `the` from the start of the next line with no space between them?...and that something similar happens when the text from lines 38 and 39 are joined together?

Reply:
Some small experimentation suggests that the text is indeed re-flowed, with line breaks removed and sequences of multiple spaces replaced by a single space.  _However_, the `{}` expression at the end of a line does indeed get joined with the text from the next line with no intervening space (which strikes me as odd).  I suggest finessing this like:
```
        <Text component={TextVariants.p}>
          Runs that have expiration date within next {EXPIRATION_DAYS_LIMIT} days
          days will appear here. These runs will be automatically removed from
          the system if left unacknowledged.
          <Button variant="link">Learn More.</Button>
        </Text>
```
That shouldn't require any explicit spaces, and I don't think it makes the lines any longer (i.e., `{" "}` is five characters, just like `days` is if you count the preceding space 🙂).

Reply:
Or, we can just merge it as it is...your call, Varshini.

Reply:
Let's merge for now. I'll check your suggestion later :grinning: